src/app/app.module.ts

```diff
@@ -36,8 +57,11 @@ import { CustomizableDemoForm } from './customizable-demo-form.component';
     MdSelectModule,
     MdSlideToggleModule,
     MdSliderModule,
+    RouterModule.forRoot(
```

Comment:
It seems to me you might as well combine this onto one line, any reason not to?

Reply:
Done (commit not pushed yet, addressing each comment one at a time)